Source/Fuse.Controls.CameraView/CameraViewBase.ScriptClass.uno

```diff
@@ -141,6 +142,28 @@ namespace Fuse.Controls
 				return new Promise<CameraFacing>().RejectWithMessage("Bad argument");
 		}
 
+		/**
+			Set CameraFocusPoint
+
+			@scriptmethod setCameraFocusPoint( x, y, cameraWidth, cameraHeight, isFocusLocked )
+
+			Returns a promise that resolves to whether or not the focus is set. Valid values are double x, double y, int cameraWidth, int cameraHeight and int isFocusLocked.
+
+				<CameraView ux:Name="Camera" />
+				<JavaScript>
+					Camera.setCameraFocusPoint(x, y, cameraWidth, cameraHeight, isFocusLocked)
+						.then(function(isSet) {  })
+						.catch(function(error) { });
+				</JavaScript>
+		*/
+		static Future<CameraFocusPoint> setCameraFocusPoint(Context context, CameraViewBase self, object[] args) 
+		{
+			if (args.Length != 5)
+				return new Promise<CameraFocusPoint>().RejectWithMessage("Arguments for CameraFocusPoint must be provided");
+
+			return self.SetCameraFocusPoint( Marshal.ToDouble(args[0]), Marshal.ToDouble(args[1]), Marshal.ToInt(args[2]), Marshal.ToInt(args[3]), Marshal.ToInt(args[4]) );
```

Comment:
I would wrap this line so it'd be easy to see all arguments even in githubs narrow view...